cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -79,22 +127,21 @@ func generateMountsModeInstall(mst *initramfsMountsState, recoverySystem string)
 		return nil
 	}
 
-	// n+1: final step: write $(tmpfs-data)/var/lib/snapd/modeenv - this
-	//      is the tmpfs we just created above
+	// 3. final step: write modeend to tmpfs data dir and disable cloud-init in
```

Comment:
```suggestion
	// 3. final step: write modeenv to tmpfs data dir and disable cloud-init in
```

Reply:
@anonymouse64 Can you please submit this in one of the followups?

Reply:
@anonymouse64 Nevermind, I did this now in feature/uc20-cross-check-mount-partitions-2